mod_reforged/config/config.nut

```diff
@@ -1,9 +1,14 @@
 ::Reforged.Config <- {
 	IsLegendaryDifficulty = true,
-	HiringCostVariance = 0.2,	// Hiring cost for recruits after gear is randomized +- this value. 0.0 = no variance. 0.2 = it varies between 80% and 120% of its original cost
-	HiringCostLuck = 200,	// Luck for rerolling hiring cost. Every 100 luck = 1 reroll. With multiple rerolls present only the closest to the original hiring cost is taken. 0 luck = to linear distribution between min and max,
-	XPOverride = false, // Is set to true during actor.onActorKilled to prevent any xp gain via player.getXP function. Then set to false afterwards. This is to do our own override of XP gain system based on damage dealt ratios.
-	VeteranPerksLevelStep = 4, // During veteran character levels, a new perk point is gained every this many levels.
+
+	Player = {
+		HiringCostVariance = 0.2,	// Hiring cost for recruits after gear is randomized +- this value. 0.0 = no variance. 0.2 = it varies between 80% and 120% of its original cost
+		HiringCostLuck = 200,	// Luck for rerolling hiring cost. Every 100 luck = 1 reroll. With multiple rerolls present only the closest to the original hiring cost is taken. 0 luck = to linear distribution between min and max,
+		XPOverride = false, // Is set to true during actor.onActorKilled to prevent any xp gain via player.getXP function. Then set to false afterwards. This is to do our own override of XP gain system based on damage dealt ratios.
+		VeteranPerksLevelStep = 4, // During veteran character levels, a new perk point is gained every this many levels.
+		DailyCostMultPerLevelRegular = 1.1,		// Exponential increase of daily cost for each level before max level
```

Comment:
I'd rather just use the `onUpdate( _properties) ` function of `character_background` to use variables defined within the background itself to adjust the `DailyCostMult` or `_properties.DailyWage` directly. That seems more contained and clean.

Reply:
I don't follow where: Do you not want these two variables to be global? Do you want them to exist inside of each character background?